src/test/java/org/opensearch/security/multitenancy/test/MultitenancyTests.java

```diff
@@ -387,4 +391,54 @@ public void testDashboardsAlias65() throws Exception {
         Assert.assertTrue(res.getBody().contains(".kibana_-900636979_kibanaro"));
     }
 
+
+    @Test
+    public void testTenantParametersSubstitution() throws Exception {
+        final Settings settings = Settings.builder()
+                .build();
+        setup(settings);
+        final RestHelper rh = nonSslRestHelper();
+
+        HttpResponse res;
+        final String url = ".kibana/_doc/5.6.0?pretty";
```

Comment:
Why `5.6.0`?

Reply:
This was from how the test were originally authored, would you like a different value in its place?

Reply:
I think it looks fine here.